CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The following is a curated list of changes in the Enact agate module, newest changes on the top.
 
 ## [unreleased]
+
+### Added
+- `agate/Marquee` new component
+- `agate/Button` marquee decorator
+- `agate/Button` tooltip decorator
+- `agate/Button` iconOnly, iconPosition, minWidth props
```

Comment:
I don't think that we share internal logic changes to app developers. So we might not need the following logs.
```
- `agate/Button` marquee decorator
- `agate/Button` tooltip decorator
```

Component names or prop names should be wrapped with ``.

```suggestion
- `agate/Button` prop `iconOnly`, `iconPosition`, and `minWidth`
- `agate/Marquee` component

### Fixed

- `agate/Button` to marquee when focused
- `agate/Button` to show a tooltip when hovered
```